bulbs/poll/models.py

```diff
@@ -126,7 +126,12 @@ def get_sodahead_data(self):
         response = requests.get(SODAHEAD_POLL_ENDPOINT.format(self.sodahead_id))
 
         if not response.ok:
-            raise SodaheadResponseError(response.text)
+            return {
```

Comment:
I think I'll stick with logging here. According to the documentation there are no errors: https://onion.sodahead.com/docs/api/polls/

Unless @spra85 you had a link to some better docs?